Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -325,7 +325,11 @@ WebIDL::ExceptionOr<GC::Ref<Document>> Document::create_and_initialize(Type type
             top_level_origin);
     }
 
-    // FIXME: 8. Let loadTimingInfo be a new document load timing info with its navigation start time set to navigationParams's response's timing info's start time.
+    // 8. Let loadTimingInfo be a new document load timing info with its navigation start time set to navigationParams's response's timing info's start time.
+    DOM::DocumentLoadTimingInfo load_timing_info;
+    // AD-HOC: The response object no longer has an associated timing info object. For now, we use response's non-standard response time property,
+    //         which represents the time that the time that the response object was created.
+    load_timing_info.navigation_start_time = navigation_params.response->response_time().nanoseconds() / 1e6;
```

Comment:
The HTML spec steps seem to be out of sync with the Fetch spec here, as the `Response` object no longer has an associated timing info object. It is important to assign *some* kind of time from the monotonic clock here so that `Performance.timeOrigin` can return a reasonable value.